securedrop/models.py

```diff
@@ -73,7 +47,7 @@ def get_one_or_else(query, logger, failure_method):
         failure_method(404)
 
 
-class Source(Base):
```

Comment:
Hm I'm not sure we want to move away from `declarative_base` - we'll want to preserve that in order to [enable autogenerated migrations for alembic](http://alembic.zzzcomputing.com/en/latest/autogenerate.html#auto-generating-migrations), which we definitely want for developer agility

Reply:
I have used this pattern to generate at least one auto migration, so I'm fairly sure this works. If it doesn't, I can swap it out when I add alembic.

Reply:
From `help(models)`:

```
    class Journalist(sqlalchemy.ext.declarative.api.Model)
     |  Method resolution order:
     |      Journalist
     |      sqlalchemy.ext.declarative.api.Model
     |      flask_sqlalchemy.model.Model
     |      __builtin__.object
```

So it looks like it still is `declarative_base` under the hood too.

Reply:
Ahh right, so we can use `db.metadata` instead of using `Base.metadata` as the target metadata for migrations. Sounds good!